.github/workflows/codecov.yml

```diff
@@ -20,12 +20,7 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - name: Checkout Sources
-        uses: actions/checkout@v4
-      - name: Fix kernel mmap rnd bits
-        # High entropy setting in GH runner images >= 20240310.1.0
-        # causes ASAN blowing up here and there:
-        # https://github.com/actions/runner-images/issues/9491
-        run: sudo sysctl vm.mmap_rnd_bits=28
+        uses: actions/checkout@v3
```

Comment:
We can probably keep v4?

Reply:
I just wanna revert the whole commit, it's fine to not updating the version for checkout action, we still use v2 some where.